lib/utils/CrossfilterHistogramPlot.ts

```diff
@@ -101,6 +104,38 @@ export function CrossfilterHistogramPlot(
 				.attr("dx", (_, i) => i === 0 ? "-0.25em" : "0.25em");
 		});
 
+	// Value under cursor label
+	const hoveredTick = svg
+		.append("g")
+		.attr("transform", `translate(0,${height - marginBottom})`)
+		.call(
+			d3
+				.axisBottom(x)
+				.tickValues([0])
+				.tickFormat(tickFormatterForBins(type, bins))
```

Comment:
Would it be useful to reuse `tickFormatterForBins(type, bins)` to format the hover value? We probably don't need all the sigfigs... also not sure how dates would show up with ~s formatter.

Reply:
Gotcha, I was wondering about a more robust way to format the different numbers. Exchanged d3.format for tickFormatterForBins, but I'm probably missing something, as the formatting doesn't improve much.